i18n/userguide/fr.po

```diff
@@ -888,36 +886,45 @@ msgid ""
 "font if available) as the current music font and trigger the {preview} to be "
 "updated."
 msgstr ""
+"En sélectionnant une ligne dans la liste, la police donnée (avec ses "
+"accolades si elles sont disponibles) sera sélectionnée comme police de "
+"musique actuelle et la {preview} sera mise à jour."
 
 #: ../../frescobaldi_app/userguide/documentfonts_music.md:30
-#, fuzzy
 msgid "Managing Music Fonts"
-msgstr "Gérer les fichiers"
+msgstr "Gestion des polices musicales"
 
 #: ../../frescobaldi_app/userguide/documentfonts_music.md:32
 msgid ""
 "Above the font list is a row with buttons to manage music font installation "
 "in the current LilyPond instance."
 msgstr ""
+"Au-dessus de la liste des polices se trouve une rangée de boutons permettant "
+"de gérer l’installation des polices de musique dans l’instance actuelle de "
+"LilyPond."
 
 #: ../../frescobaldi_app/userguide/documentfonts_music.md:35
-#, fuzzy
 msgid "Local Font Repository"
-msgstr "Aller à l’emplacement suivant"
+msgstr "Dépôt de polices local"
 
 #: ../../frescobaldi_app/userguide/documentfonts_music.md:37
 msgid ""
 "If a directory is made known as a local repository for music fonts (see "
 "{paths}) its content can easily be used as a source for font installation in "
 "arbitrary numbers of LilyPond instances. The easiest way to make use of this "
-"is to download or clone the font repository from GitHub ({notationfonts}) "
+"is to download or clone the font repository from Github ({notationfonts}) "
```

Comment:
IIRC when I skimmed through the previous PR Github was changed to GitHub. Now it's changed back to Github. GitHub is correct, but we'd better fix also the English manual.